crates/ruff_cli/src/commands/format.rs

```diff
@@ -347,6 +399,23 @@ impl Display for FormatSummary<'_> {
                     formatted += 1;
                 }
                 FormatResult::Unchanged => unchanged += 1,
+                FormatResult::Diff {
+                    path,
+                    unformatted,
+                    formatted: formatted_source,
+                } => {
+                    // Diff
```

Comment:
Nit: can probably remove this comment, it doesn't add much on top of the match guard itself (`FormatResult::Diff`).